errors.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+type ErrValidation struct {
```

Comment:
In such cases, when Error is actually a string I usually use something simpler.
```suggestion
type ErrValidation string
```

This would be the only change :smile:

Reply:
Anyways, I am glad you managed to find out quickly how to create custom errors :smiley: